src/modal/ShareModal.tsx

```diff
@@ -0,0 +1,79 @@
+import { Modal } from "./modalcomponents/Modal";
+import { ModalContentBox } from "./modalcomponents/ModalContentBox";
+import { ModalContentDescription } from "./modalcomponents/ModalContntDescription";
+import { ModalContentTitle } from "./modalcomponents/ModalTitle";
+import "../css/ShareModal.css";
+import React, { KeyboardEventHandler, MouseEventHandler } from "react";
+import kakao from "../image/kakao.png";
+import facebook from "../image/facebook.png";
+import link from "../image/link.png";
+
+interface ShareModalProps {
+  isOpen: boolean;
+  folderName: string;
+  onKakaoClick: () => void;
```

Comment:
P2: 인자가 없을 경우 아래와 같이 type을 적어주실 수 있습니다.

```suggestion
  onKakaoClick: VoidFunction;
```